web/styles/app_variables.css

```diff
@@ -67,6 +67,11 @@
     /* The legacy values here are updated via JavaScript */
     --base-font-size-px: 14px;
     --base-line-height-unitless: 1.214;
+    /* This length value is for establishing heights, possibly
+       also widths in the case of square elements, that will always
+       fit with the line-height (e.g., inline emoji, or vote counts
+       on polls) */
+    --line-height-fitted-length-px: 16px;
```

Comment:
Isn't the legacy value 17? Or does this need to be strictly less?

Reply:
In this original approach, yes, I was going for something that would be smaller than the actual line-height value.